### ISSUE TITLE
Can't build Android when iOS is enabled on package.json [Windows]

### ISSUE THREAD

**Issue by YanDevDe** [timbru31/cordova-plugin-lottie-splashscreen#187]
This is related to issue #158 :

```
Discovered platform "ios@^5.1.1" in config.xml or package.json. Adding it to the project
Der Befehl "sh" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.

Running command: sh C:\Users\Yan\Projekt\Kunden\PAC Global\pacapp\src-cordova\plugins\cordova-plugin-lottie-splashscreen\hooks\ios\update_pod_repo.sh C:\Users\Yan\Projekt\Kunden\PAC Global\pacapp\src-cordova

Error: Hook failed with error code ENOENT: C:\Users\Yan\Projekt\Kunden\PAC Global\pacapp\src-cordova\plugins\cordova-plugin-lottie-splashscreen\hooks\ios\update_pod_repo.sh
```

The issue here is that if I want to build android version, then I'm getting this error about "iOS" script file. 

I'm using windows (my main workplace) for building Android apps and I'm using MacBook for building iOS apps.

This error goes away when I'm removing "ios" from "platforms" under cordova settings at package.json. But then I have to add it again on MacBook which it annoys me a bit if we have to do that everytime.

I believe if I build apps android on windows, then the script file "update_pod_repo.sh" shouldn't be running at all. Why do we need this script on building android anyway? 😅 

Or Am I doing it wrong?

Regards,

Yan

**Comment by timbru31** [comment]
Could you share the commands you've run in order to get this issue? Adding the `ios` platform to a project on my Windows machine results in no errors when running `cordova build android`.

**Comment by YanDevDe** [comment]
I'm using this command:

```
cordova build android --release --production
```

Let me know if that doesn't appear too. 🤔

**Comment by YanDevDe** [comment]
Oh, pardon. I was actually using

```
cordova prepare
```

which is by default from vue-cli-plugin-cordova

**Comment by timbru31** [comment]
Yes, `prepare` without a platform argument prepares all platforms. I've remove the now obsolete hook anyways, published as 0.8.1-beta.1

**Comment by YanDevDe** [comment]
Ah awesome. Thank you so much. I'll try out again if I have to open the project again 👍 😄
